cli/src/commands/enrich.rs

```diff
@@ -101,6 +112,55 @@ pub struct SnykArgs {
     pub project_id: Option<String>,
 }
 
+/// Args for use with the Sbom scorecard enrichment.
+#[derive(Clone, Debug, Parser)]
+pub struct SbomScorecardArgs {
+        /// The file path to create a scorecard from
+        #[arg(long)]
+        pub sbom_file_path_1: Option<String>,
+        /// Second file path to compare with the first
+        #[arg(long)]
+        pub sbom_file_path_2: Option<String>,
+}
+
+/// Handles Sbom Scorecard enrichment commands
+struct SbomScorecardProvider {}
+
+impl SbomScorecardProvider {
+
+    async fn execute(args: &EnrichArgs) -> Result<(), Error> {
+        let mut test_sbom = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        test_sbom.push("src/services/sboms/sbom_scorecard/test_files/dropwizard.json");
+
+        match &args.sbom_scorecard_args {
+            Some(scorecard_args) => {
+                match scorecard_args {
+                    val if val.sbom_file_path_1.is_some() && val.sbom_file_path_2.is_none() => {
+                        let results = show_sbom_scorecard(val.sbom_file_path_1.clone().unwrap().to_string());
+                        match results {
+                            Ok(valid_result) => println!("\n{}", valid_result),
+                            Err(error) => println!("Failed with errors: \n{}", error),
+                        }
+                    },
+                    val if val.sbom_file_path_1.is_some() && val.sbom_file_path_2.is_some() => {
+                        let results = compare_sbom_scorecards(
+                            val.sbom_file_path_1.clone().unwrap().to_string(), 
+                            val.sbom_file_path_2.clone().unwrap().to_string()
+                        );
+                        match results {
+                            Ok(valid_result) => println!("\n{}", valid_result),
+                            Err(error) => println!("Failed with errors: \n{}", error),
+                        }
+                    },
+                    &_ => panic!("At least one path to a file is required")
+                }
+            },
+            None => panic!("A path to an Sbom file must be provided, please use --sbom-file-path-1 <path>"),
```

Comment:
My thinking was since this was for command line, we would want to stop things immediately since nothing else could be done without a proper path. However I fixed it anyways to return Scorecard Errors instead